packages/babel-plugin/src/utils/evaluate-expression/index.tsx

```diff
@@ -1,268 +1,17 @@
 import type { NodePath } from '@babel/traverse';
-import traverse from '@babel/traverse';
 import * as t from '@babel/types';
 
 import type { Metadata } from '../../types';
-import { getPathOfNode, wrapNodeInIIFE } from '../ast';
+import { getPathOfNode } from '../ast';
+import { createResultPair } from '../create-result-pair';
 import { isCompiledKeyframesCallExpression } from '../is-compiled';
-import { resolveBinding } from '../resolve-binding';
 
-import { createResultPair } from './common';
-import { traverseMemberAccessPath } from './traverse-access-path';
```

Comment:
@MonicaOlejniczak , @at-nathan I took all the methods traversing expressions, not only `traverseMemberExpression`, out of evaluate-expression. This also implied to take `createResultPair` out of it.

I know that you been reorganising the folder structure, let me know if this structure doesn't fit your vision